musl only support NPTL, so that remove that old stuff.

--- mysys/stacktrace.c.orig	2015-05-08 18:43:01.661343605 +0200
+++ mysys/stacktrace.c	2015-05-08 18:43:13.669331612 +0200
@@ -355,7 +355,7 @@ void my_print_stacktrace(uchar* stack_bo
 #endif  /* __alpha__ */
 
   /* We are 1 frame above signal frame with NPTL and 2 frames above with LT */
-  sigreturn_frame_count = thd_lib_detected == THD_LIB_LT ? 2 : 1;
+  sigreturn_frame_count = 2;
 
   while (fp < (uchar**) stack_bottom)
   {
